<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(217, 217, 217);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lb_setting">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>480</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>NanumSquareRound</family>
      <pointsize>32</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>[ 게임 설정 ]</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>250</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>NanumSquareRound</family>
      <pointsize>24</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>지뢰 탐색 횟수</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>250</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>NanumSquareRound</family>
      <pointsize>24</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>제한 시간</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_settingback">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>25</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lb_count">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>112</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>NanumSquareRound</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lb_time">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>182</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>NanumSquareRound</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>01:00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_settingsave">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>260</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>NanumSquare</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(80, 80, 80);
color: rgb(255, 255, 255);
border:none;</string>
    </property>
    <property name="text">
     <string>저 장</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_popup">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>46</x>
      <y>100</y>
      <width>388</width>
      <height>127</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="LongButton" name="btn_countup" native="true">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>102</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
   </widget>
   <widget class="LongButton" name="btn_countdown" native="true">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>102</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
   </widget>
   <widget class="LongButton" name="btn_timerdown" native="true">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>172</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
   </widget>
   <widget class="LongButton" name="btn_timerup" native="true">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>172</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
   </widget>
   <zorder>lb_setting</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>btn_settingback</zorder>
   <zorder>lb_count</zorder>
   <zorder>lb_time</zorder>
   <zorder>btn_settingsave</zorder>
   <zorder>btn_countup</zorder>
   <zorder>btn_countdown</zorder>
   <zorder>btn_timerdown</zorder>
   <zorder>btn_timerup</zorder>
   <zorder>lb_popup</zorder>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LongButton</class>
   <extends>QWidget</extends>
   <header>longbutton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
